Resources/config/controller.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="Payum\Bundle\PayumBundle\Controller\AuthorizeController" class="Payum\Bundle\PayumBundle\Controller\AuthorizeController" public="true">
```

Comment:
This config can be simplified to automatically register all controllers without having to manually specify all of them

```xml
<prototype namespace="Payum\Bundle\PayumBundle\Controller\" resource="../../Controller/*" public="true">
    <tag name="controller.service_arguments"/>
</prototype>
```

You can then also enable auto-wiring so that the right services are injected by default

```xml
<defaults autowire="true" />
```

Reply:
You can put that in project but you should not do that in reusable bundles. https://symfony.com/doc/current/bundles/best_practices.html#services